web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -135,9 +113,22 @@ export class LayerDialogComponent implements OnDestroy {
   onProjectLoaded(project: Project) {
     if (this.layerId === ':new') {
       this.layerId = this.dataStoreService.generateId();
+      const fieldId = this.dataStoreService.generateId();
+      this.fields = this.fields.push(
+        new Field(
+          fieldId,
+          1,
+          StringMap({
+            en: '',
+          }),
+          false,
+          0,
```

Comment:
Also, in Google style, constant args should be prefixed with /* argname= */.
https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments